### ISSUE TITLE
[accessibility] `title` does not label the control

### ISSUE THREAD

**Issue by Malvoz** [consbio/Leaflet.ZoomBox#19]
![control-not-accessible](https://user-images.githubusercontent.com/26493779/110104570-994e7b80-7da7-11eb-89b4-5ccaa31f61a6.png)

The `title` should be set on the `<a>` control instead of the parent `<div>`. This ensures users with screen readers can access the `title` description.

**Comment by brendan-ward** [comment]
Please consider submitting a PR to address this.
